$OpenBSD: patch-configure,v 1.13 2014/08/11 18:34:59 sthen Exp $

merged patches; cannot add new files to cvs until unlock

--- configure.orig	Sat May 10 21:27:34 2014
+++ configure	Sat May 10 22:57:55 2014
@@ -5240,7 +5240,7 @@ cat >>confdefs.h <<_ACEOF
 #define DEFAULT_NAGIOS_GROUP "$icinga_grp"
 _ACEOF
 
-INSTALL_OPTS="-o $icinga_user -g $icinga_grp"
+INSTALL_OPTS=""
 
 
 
@@ -5261,7 +5261,7 @@ fi
 
 
 
-COMMAND_OPTS="-o $command_user -g $command_grp"
+COMMAND_OPTS=""
 
 
 
@@ -5999,7 +5999,7 @@ $as_echo_n "checking for gdImagePng in -lgd (order 1).
   $as_echo_n "(cached) " >&6
 else
   ac_save_LIBS="$LIBS"
-	LIBS="-lgd -lttf -lpng -ljpeg -lz -lm $LIBS"
+	LIBS="-lgd -lpng -ljpeg -lz -lm $LIBS"
 	cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 	/* Override any gcc2 internal prototype to avoid an error.  */
@@ -6030,7 +6030,7 @@ fi
 $as_echo "yes" >&6; }
 
 			GDLIBFOUND=yep
-			GDLIBS="-lgd -lttf -lpng -ljpeg -lz -lm"
+			GDLIBS="-lgd -lpng -ljpeg -lz -lm"
 
 	else
 	  { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
@@ -7289,7 +7289,7 @@ $as_echo_n "checking for SSL libraries... " >&6; }
 		found_ssl=no
 		for dir in $ssl_lib_dir $ssl_dir /usr/lib/*-linux-gnu /usr/lib64 /usr/lib /usr/local/lib /usr/lib/ssl /usr/ssl/lib /usr/openssl/lib /usr/pkg/lib  /usr/freeware/lib/openssl; do
 			ssllibdir="$dir"
-			if test -f "$dir/libssl.so"; then
+			if test -f "$dir/libssl.a"; then
 				found_ssl=yes
 			        break
 			fi
--- base/logging.c.orig	Mon Jul 28 23:13:36 2014
+++ base/logging.c	Mon Jul 28 23:12:30 2014
@@ -306,7 +306,7 @@ int write_to_log_with_host_service(char *buffer, unsig
 	strip(buffer);
 
 	/* write the buffer to the log file */
-	fprintf(fp, "[%lu] %s\n", log_time, buffer);
+	fprintf(fp, "[%lld] %s\n", (long long)log_time, buffer);
 	fflush(fp);
 
 #ifdef USE_EVENT_BROKER
